.devcontainer/devcontainer.json

```diff
@@ -4,27 +4,35 @@
         "../scripts/ci/docker-compose/devcontainer.yml",
         "../scripts/ci/docker-compose/devcontainer-sqlite.yml"
     ],
-    "settings": {
-        "terminal.integrated.defaultProfile.linux": "bash"
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "terminal.integrated.defaultProfile.linux": "bash"
+            },
+            "extensions": [
+                "ms-python.python",
+                "ms-python.vscode-pylance",
+                "mtxr.sqltools",
+                "mtxr.sqltools-driver-pg",
+                "rogalmic.bash-debug",
+                "ms-azuretools.vscode-docker",
+                "dbaeumer.vscode-eslint",
+                "ecmel.vscode-html-css",
+                "timonwong.shellcheck",
+                "redhat.vscode-yaml",
+                "rogalmic.bash-debug",
+                "tamasfe.even-better-toml",
+                "ms-python.debugpy",
+                "charliermarsh.ruff",
+                "ms-python.mypy-type-checker"
+            ]
+        }
     },
-    "extensions": [
-        "ms-python.python",
-        "ms-python.vscode-pylance",
-        "mtxr.sqltools",
-        "mtxr.sqltools-driver-pg",
-        "rogalmic.bash-debug",
-        "ms-azuretools.vscode-docker",
-        "dbaeumer.vscode-eslint",
-        "ecmel.vscode-html-css",
-        "timonwong.shellcheck",
-        "redhat.vscode-yaml",
-        "rogalmic.bash-debug"
-    ],
     "service": "airflow",
     "forwardPorts": [8080, 5555, 5432, 6379],
     "workspaceFolder": "/opt/airflow",
     // for users who use non-standard git config patterns
     // https://github.com/microsoft/vscode-remote-release/issues/2084#issuecomment-989756268
-    "initializeCommand": "cd \"${localWorkspaceFolder}\" && git config --local user.email \"$(git config user.email)\" && git config --local user.name \"$(git config user.name)\"",
+    // "initializeCommand": "cd \"${localWorkspaceFolder}\" && git config --local user.email \"$(git config user.email)\" && git config --local user.name \"$(git config user.name)\"",
```

Comment:
How did you "fix" the initializeCommand? I just see it commented-out.
If it is not needed anymore, then please delete the line.
Otherwise I'd assume you forgot to push the fix?

Reply:
Maybe it would fix for someone else, for me it made problems.

Reply:
Also view [this](https://github.com/apache/airflow/pull/42400#issuecomment-2393139913).

Reply:
Removing it fixed the problem. But what if people would encounter the problem described there? I think it's good to leave it like that, so they could at least know what to do if they encounter a problem.

Reply:
Are you sure it does not break anything else?